app/src/main/java/org/feature/fox/coffee_counter/ui/transaction/TransactionActivity.kt

```diff
@@ -0,0 +1,24 @@
+package org.feature.fox.coffee_counter.ui.transaction
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.feature.fox.coffee_counter.ui.theme.CoffeeCounterTheme
+
+
+@AndroidEntryPoint
+class TransactionActivity : ComponentActivity() {
+
+    private val transactionViewModel: TransactionViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            CoffeeCounterTheme {
+                HistoryView()
+            }
+        }
+    }
+}
```

Comment:
We are currently using the CoreActivity for all four views. Therefore i suggest removing this activity for now.
This is due to the use of the NavigationBar that automatically leads to all 4 basic views.